src/codecs.js

```diff
@@ -1,3 +1,7 @@
+import window from 'global/window';
+const videoSupportRegex = RegExp('^(avc[13]|vp09|av01)');
```

Comment:
I know we support `avc1`.  It was also listed in `parseCodecs` that we support the `avc3`, `vp09`, `av01`.